cli/formatter/formatter.go

```diff
@@ -0,0 +1,51 @@
+package formatter
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+)
+
+// Format key to output
+const (
+	TableFormat = "table"
+	RawFormat   = "raw"
+)
+
+// IsTable to verify if table or raw
+func IsTable(format string) bool {
+	return strings.HasPrefix(format, TableFormat)
+}
+
+// PreFormat is to format the format option
+func PreFormat(format string) string {
+	if IsTable(format) {
+		format = format[len(TableFormat):]
+		//cut the space
+		format = strings.Trim(format, " ")
+		//input by cmd of "\t" is "\\t"
+		replace := strings.NewReplacer(`\t`, "\t", `\n`, "\n")
```

Comment:
could you explain the logic here? why we need this? thanks.

Reply:
format = strings.Trim(format, " ")
user input "table        {{.ID}}         "------ no trim for space:the output will have so many space
replace := strings.NewReplacer(`\t`, "\t", `\n`, "\n")
seems in the cobra command. the input of "\" will be consider as `\`  not "\".
so need to transfer it.